moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -197,6 +197,7 @@ MotionPlanningFrame::MotionPlanningFrame(MotionPlanningDisplay* pdisplay, rviz::
 
 MotionPlanningFrame::~MotionPlanningFrame()
 {
+  delete ui_;
```

Comment:
Is MotionPlanningFrame copyable or movable? If it is either, this could lead to double deletes if the ui_ pointer ends up being copied (the default move constructor also copies pointers, if it exists). It would be safer to wrap the object in a unique_ptr.

Reply:
At least it is not intended to be copied ;-)

Reply:
Right, then I submit to your judgment wether we should keep it like this or put it in a `std::unique_ptr` :)

Reply:
Nevermind, just noticed the deleted copy constructor :)

Reply:
I added it due to your comment.